program/swap-two-numbers/swap-two-numbers.py

```diff
@@ -0,0 +1,5 @@
+a=int(input())
+b=int(input())
+a,b=b,a
```

Comment:
Also the solution of swapping two numbers without a third variable (just for others..)